tests/realtime/realtime.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	"kubevirt.io/kubevirt/pkg/libvmi"
 	libvmici "kubevirt.io/kubevirt/pkg/libvmi/cloudinit"
-	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
 
```

Comment:
nit: remove newline

Reply:
The newline separates production from e2e tests packages.